test/test_artifacts/v2/maxdome_data_explorer.test.Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG COSMOS_IMAGE
```

Comment:
Can you update this to 
```
ARG SAGEMAKER_DISTRIBUTION_IMAGE
FROM $SAGEMAKER_DISTRIBUTION_IMAGE

Reply:
done, https://github.com/aws/sagemaker-distribution/pull/373/commits/dcda492365a6584bf0d00d8b4c5f152534125837

Do we also need to update the other ones?

* https://github.com/aws/sagemaker-distribution/blob/2.0.0-beta/test/test_artifacts/v2/maxdome_jupyter_server_extension.test.Dockerfile#L1-L2

* https://github.com/aws/sagemaker-distribution/blob/2.0.0-beta/test/test_artifacts/v2/maxdome_jupyter_session_manager.test.Dockerfile#L1-L2

Reply:
Ah yeah looks like it. Used to be COSMOS_IMAGE but we changed all the ones on `main` recently

Reply:
got it, i belive this is in the process of being reverted so not doing it in this PR.